content/CORE/UIReference/TopMenu/UIPreferences.md

```diff
@@ -12,80 +16,82 @@ aliases: /core/administration/uipreferences/
-| Prefer buttons with icons only | checkbox | Preserve screen space with icons and tooltips instead of text labels. |
-| Enable Password Toggle | checkbox | When set, an eye icon appears next to password fields. Clicking the icon reveals the password. |
-| Reset Table Columns to Default | checkbox | Reset all tables to display the system default columns. |
-| Retro Logo | checkbox | Revert branding back to FreeNAS. |
-| Reset All Preferences to Default | checkbox | Reset all user preferences to their default values. (Custom themes are preserved) |
-| UPDATE PREFERENCES | button | Apply the current checkbox settings to the web interface. |
+| **Choose Theme** | dropdown list | Choose a preferred theme. Prebuilt and custom created themes are visible here. |
+| **Prefer buttons with icons only** | checkbox | Preserve screen space with icons and tooltips instead of text labels. |
+| **Enable Password Toggle** | checkbox | Select to make an eye icon appear next to password fields. Click the icon to reveal the password. |
+| **Reset Table Columns to Default** | checkbox | Reset all tables to display the system default columns. |
+| **Retro Logo** | checkbox | Revert branding back to FreeNAS. |
+| **Reset All Preferences to Default** | checkbox | Reset all user preferences to their default values. (Custom themes are preserved) |
+| **UPDATE PREFERENCES** | button | Apply the current checkbox settings to the web interface. |
 
 ## Manage Custom Themes
 
 | Option | Style | Description |
 |--------|-------|-------------|
-| theme name (variable) | checkbox | Each custom theme is listed. |
-| DELETE SELECTED | button | Removes each selected custom theme from the system. |
-| CREATE NEW THEME | button | Opens the theme creation screen. |
+| **theme name (variable)** | checkbox | Each custom theme is listed. |
+| **DELETE SELECTED** | button | Removes each selected custom theme from the system. |
+| **CREATE NEW THEME** | button | Opens the theme creation screen. |
 
 ## Custom Theme Editor
 
-![Preferences Custom Theme](/images/CORE/12.0/PreferencesCustomTheme.png "Custom UI Theme")
+![Preferences Custom Theme](/images/CORE/13.0/PreferencesCustomTheme.png "Custom UI Theme")
 
 ### Create Theme
 
 | Option | Style | Description |
 |--------|-------|-------------|
-| Load colors from existing theme | drop down | Imports the settings from the chosen theme into the **Create Theme** and **Preview** tabs. |
-| GENERAL | tab | Contains the primary options for the new theme. |
-| COLORS | tab | Contains options for setting custom color values. |
-| PREVIEW | tab | Controls the **Preview** widget. |
+| **Load colors from existing theme** | dropdown list | Imports the settings from the chosen theme into the **Create Theme** and **Preview** tabs. |
+| **GENERAL** | tab | Contains the primary options for the new theme. |
+| **COLORS** | tab | Contains options for setting custom color values. |
+| **PREVIEW** | tab | Controls the **Preview** widget. |
 
```

Comment:
Same thing as previous tables, we are standardizing on not using a column for the type of UI element, so remove and fix the descriptions. For  tabs, you can use "Click to display the **GENERAL** tab with the primary options for a new theme." or some variation on the idea. With a dropdown list, use something like "Select the theme option from the dropdown list to import settings into the **Create Theme"" and **Preview** tabs.